scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml

```diff
@@ -517,6 +517,27 @@ TSSOP-24_4.4x7.8mm_P0.65mm:
   num_pins_x: 0
   num_pins_y: 12
 
+TSSOP-24-1EP_4.4x7.8mm_P0.65mm_EP3.2x5mm:
+  size_source: 'JEDEC MO-153 Var AD https://www.jedec.org/document_search?search_api_views_fulltext=MO-153'
```

Comment:
This package is underspecified by this size source, MO-153 only specifies minimum EP size.
Please use ST documentation.